src/main/php/inject/Injector.class.php

```diff
@@ -232,12 +231,12 @@ public function newInstance(XPClass $class, $named= []) {
   /**
    * Inject members of an instance
    *
-   * @param   lang.Generic $instance
-   * @return  lang.Generic
+   * @param   var $instance A value object
+   * @return  var
    * @throws  inject.ProvisionException
```

Comment:
It might be nice to have an object type which does instance of checks via `is_object()` - but this would need support in the framework.